### DISCUSSION TITLE
请问DagreLayout布局节点怎么居中展示

### DISCUSSION THREAD

**Discussion by rochaie** [antvis/X6#2165] [Q&A]
api好像只有靠左和靠右对齐

**Comment by NewByVector** [comment]
布局之后整体调用 `graph.centerContent()` 来居中就可以。

**Reply by rochaie** [reply]
我指的是align属性里面的对齐，比如说第一层节点只有一个，然后第二层有多个，第一层的节点居中对齐
![image](https://user-images.githubusercontent.com/48020484/171318079-4ef09c20-1897-47fc-9dbe-290f7f28f596.png)

**Reply by NewByVector** [reply]
`align` 不设置就是居中对齐。
